src/settings.py

```diff
@@ -57,7 +57,7 @@ class GlobalSettings:
     random_options_max_tries = 100
 
     # SeSamE parameters
-    task_planning_heuristic = "hadd"  # hadd or hmax
+    task_planning_heuristic = "hadd"  # hadd or hmax or hff
```

Comment:
based on your comment, let's just make this env-specific for the sake of progress? i'd be fine with that

Reply:
predicate invention envs are all using hadd, while amber project is using hff, so it's not an issue paper-wise

Reply:
sounds good